.github/workflows/reset-azure-environment.yml

```diff
@@ -0,0 +1,193 @@
+          database.
+      service-objective-name:
+        required: false
+        type: string
+        default: S0
+        description: The name of the service objective to scale the database to, e.g. B, S1, S4, P4, P6.
+      blob-container-exclude-list:
+        required: false
+        type: string
+        default: "@()"
+        description: >
+          PowerShell string array with the name of the excluded Blob Container(s), e.g.
+          "@(\"not-media\")" or "@(\"not-media\", \"bad-container\")". The parameter must be a PowerShell string array.
+          When valid values are provided for "blob-container-include-list", then "blob-container-exclude-list" is not
+          taken into consideration.
+      blob-container-include-list:
+        required: false
+        type: string
+        default: "@(\"media\")"
+        description: >
+          PowerShell string array with the name of the included Blob Container(s), e.g.
+          "@(\"media\")" or "@(\"media\", \"cool-container\")". The parameter must be a PowerShell string array. When
+          valid values are provided, it cancels out "blob-container-exclude-list".
+      folder-exclude-list:
+        required: false
+        type: string
+        default: "@(\"RecipeJournal\", \"`$`$`$ORCHARD`$`$`$.`$`$`$\")"
+        description: >
+          PowerShell string array with the name of the excluded folder(s), e.g.
+          "@(\"RecipeJournal\")" or "@(\"RecipeJournal\", \"BadFolder\")". The parameter must be a PowerShell string
+          array. When valid values are provided for "folder-include-list", then "folder-exclude-list"
+          is not taken into consideration.
+      folder-include-list:
+        required: false
+        type: string
+        default: "@()"
+        description: >
+          PowerShell string array with the name of the included folder(s), e.g.
+          "@(\"NotRecipeJournal\")" or "@(\"NotRecipeJournal\", \"CoolFolder\")". The parameter must be a PowerShell
+          string array. When valid values are provided, it cancels out "folder-exclude-list".
+
+jobs:
+  reset-azure-environment:
+    runs-on: ${{ inputs.machine-type }}
+    name: Reset Azure Environment
+    defaults:
+      run:
+        shell: pwsh
+    timeout-minutes: ${{ inputs.timeout-minutes }}
+    steps:
+    - name: Login to Azure
+      # v1.4.5
+      uses: azure/login@24848bc889cfc0a8313c2b3e378ac0d625b9bc16
+      with:
+        creds: ${{ secrets.AZURE_APP_SERVICE_RESET_SERVICE_PRINCIPAL }}
+        enable-AzPSSession: true
+
+    - name: Initialize PowerShell modules
+      uses: Lombiq/Infrastructure-Scripts/.github/actions/initialize@issue/NEST-338
+
+    - name: Stop Web App Slot
+      run: |
+        Stop-AzureWebAppSlot `
+          -ResourceGroupName ${{ inputs.resource-group-name }} `
+          -WebAppName ${{ inputs.app-name }} `
+          -SlotName ${{ inputs.destination-slot-name }}
+
+    - name: Replace Media
```

Comment:
This too unexpectedly long time here: https://github.com/Lombiq/DotNest-Tenants-Core/actions/runs/3404808941/attempts/3 While attempt 2 was normal: https://github.com/Lombiq/DotNest-Tenants-Core/actions/runs/3404808941/attempts/2

Reply:
It is because of the frequent runs. I tested it multiple times. When I ran it only 1-2 minutes after the other finished, it always resulted in 20-30 minutes long Replace Media step. If you wait 5-10 minutes after a run it will average around 7-10 minutes for the replace media step.

Reply:
I see, that's OK then.